.github/workflows/daily_check.yml

```diff
@@ -48,9 +48,11 @@ jobs:
         run: |
           pip3 install .
           pip3 freeze > version_specific_results/pypi_performance_37_${{needs.Check_Pyccel_Version.outputs.new_version }}_requirements.txt
+      - name: Generate pyccel configs
+        uses: ./.github/actions/generate_pyccel_config
       - name: Benchmark
         run: |
-          python benchmarks/run_benchmarks.py --pyperf --verbose
+          python benchmarks/run_benchmarks.py --pyperf --verbose --pyccel-config-files benchmarks/pyccel_fortran.json benchmarks/pyccel_c.json --pythran-config-files benchmarks/pythran.config
```

Comment:
Why not `cd`ing into the `benchmarks` directory before running the script? This would shorten the command considerably

Reply:
Fixed 1760478